preview/FS-1068-open-type-declaration.md

```diff
@@ -291,33 +242,6 @@ Try to resolve `member-ident` to one of the following, in order:
 That is to say, we will resolve methods over properties in the context of opening static classes or static members extending a static class.
 
 
-### No opening static classes with generic parameter instantiations
-
-C# allows for opening static classes with generic parameters like this:
-
-```csharp
-using static MyStaticClass<int>;
-
-M(12);
-```
-
-When opening different generic instantiations of `MyStaticClass`, the specialized members are viewed as overloaded methods:
-
-```csharp
-using static MyStaticClass<int>;
-using static MyStaticClass<string>;
-
-M(12); // This is one overload
-M("hello"); // This is another overload
-```
-
-This appears barely documented in the C# design docs. The preview version of the F# feature explicitly does not allow this: `open type` is only allowed on non-generic static classes.
-
-From an interop perspective, opening a generic static class may be required as some APIs may use generic type parameters on the static class, forcing users to open these with a concrete substitution when using them.
-
-We will not do this for F#.
-
-
 ## Compatibility
```

Comment:
Should have a section on tooling compatibility here. What are implications in tooling for this change, and what needs/needed to be done to ensure a good experience?